cmake/Modules/MacroQbtCXXCompilerAndModeCheck.cmake

```diff
@@ -0,0 +1,19 @@
+function(qbt_cxx_compiler_and_mode_check QBITTORRENT_MINIMUM_CXX_STANDARD)
```

Comment:
you can select a much shorter name for the argument, for example "_std"

Reply:
fixed